src/providers/pact-verifier.provider.ts

```diff
@@ -1,8 +1,10 @@
 import { Provider } from '@nestjs/common';
 import { Verifier } from '@pact-foundation/pact';
 import { PactModuleProviders } from '../common/pact-module-providers.enum';
+import { PactProviderOptions } from '../interfaces/pact-provider-module-options.interface';
 
 export const PactVerifierProvider: Provider<Verifier> = {
   provide: PactModuleProviders.PactVerifier,
-  useFactory: () => new Verifier(),
+  useFactory: (config: PactProviderOptions) => new Verifier(config),
```

Comment:
I think this finally fixes https://github.com/pact-foundation/nestjs-pact/issues/7 !

Reply:
we should have probably done it in smaller increments but this pr solves several issues :)